### PR TITLE
Responsive charts on resize

### PR DESCRIPTION
**Pull request by Elkrival**
When the screen is being resized dynamically the bar chart does not resize, this pr fixes this issue. By setting the width of the responsive container from 100 to 99% fixes the issue.

### CODE REVIEW

File: views/components/BarGraph/index.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const BarGraph = ({ graph }) => {
   }
 
   return (
-    <ResponsiveContainer width="100%" height={500}>
+    <ResponsiveContainer width="99%" height={500}>
```
Comment (mikestone14) [inline views/components/BarGraph/index.jsx]: interesting, any idea why that is the case? Do you have any reference material like a stack overflow post or documentation link?

Comment (Elkrival) [reply views/components/BarGraph/index.jsx]: For version 2.9 of recharts the 99% setting was a hacky solution to this problem, with the latest version of recharts 2.10 this issue should have gone away. 
https://github.com/recharts/recharts/issues/172

I've updated the library and the issue is fixed [f8a91b6](https://github.com/AMP-SCZ/dpdash/pull/599/commits/f8a91b625c0f5138f0556f043e0b44015e8c11c4)

File: .nvmrc
Comment (JackVCurtis) [inline .nvmrc:1]: Why was this deleted?

Comment (Elkrival) [reply .nvmrc:1]: The file sets the local npm and node engine and we are now developing using docker. This warning about the engine came up when I installed the package.

**Review by JackVCurtis — APPROVED** [review]
:shipit:

**Pull request merged**